test/integration/test_multiple_model_endpoint.py

```diff
@@ -55,19 +55,24 @@ def modulevolume():
 
 @pytest.fixture(scope='module', autouse=True)
```

Comment:
Do we also want to update the version in test setup(tox.ini)?

Reply:
Good catch! updated